pkg/descheduler/strategies/lownodeutilization_test.go

```diff
@@ -561,11 +561,11 @@ func TestValidateStrategyConfig(t *testing.T) {
 
 		if validateErr == nil || testCase.errInfo == nil {
 			if validateErr != testCase.errInfo {
-				t.Errorf("expected validity of strategy config: thresholds %#v targetThresholds %#v\nto be %v but got %v instead",
+				t.Errorf("expected validity of strategy config: thresholds %#v targetThresholds %#v\n to be %v but got %v instead",
```

Comment:
Do you think the `\n` is needed? Since you are changing this error message maybe it would be a good idea to remove the `\n` too.

Reply:
It seems to me `\n` is not needed, because the error message is not huge that it needs to be displayed in a new line.
so let's remove it